homeassistant/components/recorder/__init__.py

```diff
@@ -230,6 +231,7 @@ def purge_ticker(event):
 
             self.queue.task_done()
 
+    @asyncio.coroutine
```

Comment:
If we mark it as a co-routine, shouldn't we also then register it using `hass.bus.listen_async` instead of `hass.bus.listen` ?